analyzers/src/SonarAnalyzer.Common/Extensions/ITypeSymbolExtensions.cs

```diff
@@ -21,13 +21,12 @@
 using System.Collections.Immutable;
 using System.Linq;
 using Microsoft.CodeAnalysis;
+using SonarAnalyzer.Helpers;
 
-namespace SonarAnalyzer.Helpers
+namespace SonarAnalyzer.Extensions
 {
-    internal static class TypeHelper
+    internal static class ITypeSymbolExtensions
     {
-        #region TypeKind
-
         public static bool IsInterface(this ITypeSymbol self) =>
```

Comment:
Not to repeat the logic I suggest calling Is() on self with the correct typekind for the 3 checks: IsStruct, IsInterface, IsClass